src/Common/src/Interop/Windows/kernel32/Interop.CopyFile.Uap.cs

```diff
@@ -1,27 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
@JeremyKuhne, can you confirm that these IO-related interop files are no longer needed?